### ISSUE TITLE
UA 50: Description fields that contained escaped out semi-colon broken on to new lines

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#455]
### Descriptive Summary

Just happened upon this issue while looking at items in UA 50.  It looks like, upon ingest, the system ignored the escaped our semi-colons (\;) appearing in a description field and instead interpreted them as line breaks.

We want to reinsert the semi-colons, remove the backslashes, remove the line breaks, and present the text in the correct order.

### Acceptance Criteria

This is what done looks like:

* [ ] Semi-colons reinserted into descriptions
* [ ] Backslashes and line breaks removed
* [ ] Descriptive text appears in correct order

### Expected Behavior 

That the system had interpreted the escaped characters correctly.  How descriptive text should read (note: this example contains two description fields):
    
**Description 1:** Unidentified donor. 
**Description 2:** Inventory note: "General campus; Science Library, interior; and gifts for library: printing press, memorial to Vietnamese children, instument, necklace, sculpture"

### Actual Behavior

**This is how things look now:**
    and gifts for library: printing press, memorial to Vietnamese children, instument, necklace, sculpture"
    Unidentified donor.
    Inventory note: "General campus\
    Science Library, interior\

![Screen Shot 2021-03-03 at 10 07 04 AM](https://user-images.githubusercontent.com/32003407/109866172-0d065200-7c09-11eb-9adf-fc6f32906c7e.png)

https://digitalcollections.library.ucsc.edu/records/2z10wr692

**Comment by rmjaffe** [comment]
Here are a couple more examples: 
https://digitalcollections.library.ucsc.edu/records/2j62s632j
https://digitalcollections.library.ucsc.edu/records/7p88ch98x

**Comment by rmjaffe** [comment]
@snehagunduraoUL @rschwab Do we think this will require a roundtrip to correct or is it something that can be fixed another way?

**Comment by rschwab** [comment]
This could be done with either a round trip, or editing individual works manually. Example work edit page: https://digitalcollections.library.ucsc.edu/concern/works/2z10wr692/edit?locale=en

**Comment by rmjaffe** [comment]
There are too many of these so I don't think editing individual works is a practical option -- just tracking down all the works with busted descriptions would be a challenge.  I'm just bummed that it will require a roundtrip (what I feared).  As a ton of editing has been done on the metadata records in the DAMS, I can't reuse the original ingest sheet.  The descriptions are going to be a #%@* mess upon export.  Ick.

**Comment by rschwab** [comment]
If descriptions haven't been edited post-ingest, you could export the collection, remove all the description columns, then add in the original description column from the ingest sheet.

**Comment by rmjaffe** [comment]
Yeah, totally.  Unfortunately it's not clear where Frank added names or made edits to titles, descriptions or other elements of the records ... :(

**Comment by rmjaffe** [comment]
When we export the date we'll see how bad or not bad it is.
